pkg/endpoint/bpf.go

```diff
@@ -587,14 +587,30 @@ func (e *Endpoint) runPreCompilationSteps(regenContext *regenerationContext) (pr
 	stats := &regenContext.Stats
```

Comment:
nit: little typo in your commit message `to avoid warnign logs.` --> `to avoid warning logs.`

Reply:
Fixed, thanks!